report/01_Introduction.md

```diff
@@ -1,10 +1,15 @@
-# Report
+# Devoops Project Report
 
-This report details our project done February to May in 2020 during the DevOps project.
+This report details our project done February to May in 2020 during the course "DevOps, Software Evolution and Software Maintenance" at IT University of Copenhagen.
```

Comment:
Ret "our" til noget andet.

Reply:
Using the word "our" is not a problem by itself. 
I can reword if you feel very strongly about it, but in this case i would find it appropriate as it it is OUR project.

Reply:
Also in that case a suggestion would be nice, because i can't find a better wording then our.